etc/picongpu/hemera-hzdr/defq.tpl

```diff
@@ -104,6 +105,6 @@ export OMPI_MCA_io=^ompio
 
 if [ $? -eq 0 ] ; then
   # Run PIConGPU
-  source !TBG_dstPath/tbg/handleSlurmSignals.sh mpiexec --bind-to none !TBG_dstPath/tbg/cpuNumaStarter.sh \
+  source !TBG_dstPath/tbg/handleSlurmSignals.sh mpiexec -np !TBG_tasks --bind-to none !TBG_dstPath/tbg/cpuNumaStarter.sh \
```

Comment:
In my opinion, this change points to the fact that mpiexec is not working correctly together with SLURM. The variables passed in the job script are not taken into account by MPI.
It looks like MPI is not compiled against SLURM.

Reply:
It works on N full nodes without it as well. But for e.g. 6 tasks and 4 tasks per node on v100 partition mpiexec by default incorrectly launches 8 processes. I think it supports your idea